src/AVFoundation/Enums.cs

```diff
@@ -706,84 +998,185 @@ public enum AVContentAuthorizationStatus : long {
+#else
+		[Mac (10, 15)]
+#endif
 		Opportunistic = 2,
 	}
 
+#if NET
+	[SupportedOSPlatform ("maccatalyst14.0")]
+	[SupportedOSPlatform ("macos10.15")]
+	[SupportedOSPlatform ("ios10.0")]
+	[UnsupportedOSPlatform ("tvos")]
+#else
 	[Introduced (PlatformName.MacCatalyst, 14, 0)]
-	[NoTV, NoWatch, Mac (10,15), iOS (10,0)]
+	[NoTV]
+	[NoWatch]
+	[Mac (10, 15)]
+	[iOS (10, 0)]
+#endif
 	[Native]
-	public enum AVCaptureColorSpace : long
-	{
+	public enum AVCaptureColorSpace : long {
 		Srgb = 0,
 		P3D65 = 1,
+#if NET
+		[SupportedOSPlatform ("maccatalyst14.1")]
+		[SupportedOSPlatform ("ios14.1")]
+		[UnsupportedOSPlatform ("macos")]
+		[UnsupportedOSPlatform ("tvos")]
+#else
 		[Introduced (PlatformName.MacCatalyst, 14, 1)]
-		[iOS (14,1)][NoMac]
+		[iOS (14, 1)]
+		[NoMac]
+#endif
 		HlgBT2020 = 2,
 	}
 
-	[iOS (8,0)]
+#if NET
+	[SupportedOSPlatform ("ios8.0")]
+#else
+	[iOS (8, 0)]
+#endif
 	[Native]
-	public enum AVMusicTrackLoopCount : long
-	{
+	public enum AVMusicTrackLoopCount : long {
 		Forever = -1
 	}
 
-	[Watch (6,0), iOS (10,0), TV (10,0), Mac (10,12)]
+#if NET
+	[SupportedOSPlatform ("ios10.0")]
+	[SupportedOSPlatform ("tvos10.0")]
+	[SupportedOSPlatform ("macos10.12")]
+#else
+	[Watch (6, 0)]
+	[iOS (10, 0)]
+	[TV (10, 0)]
+	[Mac (10, 12)]
+#endif
 	[Native]
-	public enum AVPlayerTimeControlStatus : long
-	{
+	public enum AVPlayerTimeControlStatus : long {
 		Paused,
 		WaitingToPlayAtSpecifiedRate,
 		Playing
 	}
 
-	[NoWatch, NoTV, iOS (10,0), Mac (10,15)]
+#if NET
+	[SupportedOSPlatform ("ios10.0")]
+	[SupportedOSPlatform ("macos10.15")]
+	[UnsupportedOSPlatform ("tvos")]
+#else
+	[NoWatch]
+	[NoTV]
+	[iOS (10, 0)]
+	[Mac (10, 15)]
+#endif
 	[Native]
-	public enum AVAudioSessionIOType : long
-	{
+	public enum AVAudioSessionIOType : long {
 		NotSpecified = 0,
 		Aggregated = 1,
 	}
 
-	[Watch (6,0), iOS (10,0), TV (10,0), Mac (10,12)]
+#if NET
+	[SupportedOSPlatform ("ios10.0")]
+	[SupportedOSPlatform ("tvos10.0")]
+	[SupportedOSPlatform ("macos10.12")]
+#else
+	[Watch (6, 0)]
+	[iOS (10, 0)]
+	[TV (10, 0)]
+	[Mac (10, 12)]
+#endif
 	[Native]
 	public enum AVPlayerLooperStatus : long {
 		Unknown,
 		Ready,
 		Failed,
 		Cancelled
 	}
-	
-	[NoiOS, TV (10,0), NoWatch, NoMac]
+
+#if NET
+	[SupportedOSPlatform ("tvos10.0")]
+	[UnsupportedOSPlatform ("ios")]
+	[UnsupportedOSPlatform ("macos")]
+#else
+	[NoiOS]
+	[TV (10, 0)]
+	[NoWatch]
+	[NoMac]
+#endif
 	[Native]
 	public enum AVContentProposalAction : long {
 		Accept,
 		Reject,
 		Defer
 	}
 
-	[NoiOS, TV (10,0), NoWatch, NoMac]
+#if NET
+	[SupportedOSPlatform ("tvos10.0")]
+	[UnsupportedOSPlatform ("ios")]
+	[UnsupportedOSPlatform ("macos")]
+#else
+	[NoiOS]
+	[TV (10, 0)]
+	[NoWatch]
+	[NoMac]
+#endif
 	[Native]
 	public enum AVPlayerViewControllerSkippingBehavior : long {
 		Default = 0,
 		SkipItem
 	}
 
-	[TV (10,2), Mac (10,12,4), iOS (10,3), Watch (7,0)]
+#if NET
+	[SupportedOSPlatform ("tvos10.2")]
+	[SupportedOSPlatform ("macos12.4")]
```

Comment:
Should this be MacOS 10.12.4? (I think the min supported is MacOS 10.14)

Reply:
Yes, I just caught this in my AppKit work locally. I'll have to fix the generator and re-run here.